### ISSUE TITLE
Get familiar with ggplot

### ISSUE THREAD

**Issue by VCF** [cherrypi/Science-Fair_2019#11]
R has "built-in" plotting capabilities via `plot` and some other methods. However, they're considered fairly rudimentary, and are usually used to quickly glance at your data. For publication-quality graphics there's a great package called `ggplot2`. It can be confusing at first, but it's not too bad once you get over the initially weird syntax.

[This is a good introduction][intro] to the package. Try out the examples, and then see if you can adapt your data to what you've learned.

* Focus on scatter plots for the moment, we'll deal with your interest in radar plots later
* Don't worry about formatting - we can get to that later, too.
* Note that the examples use sample data sets that come with R (like `midwest`)
* ggplot has mechanisms that let you easily integrate "fits" to your data - these are the `geom_smooth()` components you'll see in the examples. These can be very valuable in highlighting trends or relationships.
* As you work with your data, keep an eye out both for interesting patterns, and patterns that you thought you'd see _but did not_. Negative observations can be as important as positive ones!

[intro]: http://r-statistics.co/Complete-Ggplot2-Tutorial-Part1-With-R-Code.html#2.%20Making%20a%20simple%20scatter%20plot

**Comment by cherrypi** [comment]
My terminal claims it can't find the ggplot2 package:

`Error in library(ggplot2) : there is no package called ‘ggplot2’
`

**Comment by cherrypi** [comment]
Otherwise, I think I'm pretty much good.

**Comment by VCF** [comment]
That means you haven't installed it yet. Try:
```r
install.packages("ggplot2")
## And remember to use any package, your code should pull it in with library, which
## is generally put at the top of your code.
library("ggplot2")
```

**Comment by VCF** [comment]
You have a typo in 5393e8a - look at the error that's being thrown to get a hint of where it is.

**Comment by VCF** [comment]
Ok, good, fixed in  ce5c87d. It looks like you have a good grasp of basic plotting!
